src/wr_led_matrix/package.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>wr_led_matrix</name>
+  <version>0.0.0</version>
+  <description>The wr_led_matrix package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="parallels@todo.todo">parallels</maintainer>
```

Comment:
Maybe throw your name in here just to know you were the main developer.  Otherwise, you might not be the first person we would go to with questions.

Reply:
This is still not changed, which may be okay, but you should comment saying that you want it this way to resolve this.

Reply:
I must have forgot to save the file, but the new commit has my email and github username